backends/p4fmt/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+set(FMT_SRCS
+    p4fmt.cpp
+    options.cpp
+    )
```

We do not have a CMake style but I would fix this indentation here. 
```suggestion
)
```